sdk/storage/storage-blob/samples/typescript/basic.ts

```diff
@@ -108,7 +94,7 @@ async function main() {
 async function streamToString(readableStream: NodeJS.ReadableStream) {
```

Comment:
I remembered helper methods like `toStream`, `toStream` will be integrated into SDK right? @jeremymeng